fission-core/library/Fission/Unit/Prefix/Types.hs

```diff
@@ -0,0 +1,116 @@
+module Fission.Unit.Prefix.Types
+  ( Unity (..)
+  , Deci  (..)
+  , Centi (..)
+  , Milli (..)
+  , Micro (..)
+  , Pico  (..)
+  ) where
```

Comment:
This module is me getting tired of going "is this in seconds, milliseconds, microseconds, or nanoseconds". Just tracking it with the types, and having it do order-of-magnitude conversions for us.